Makefile

```diff
@@ -1,94 +1,75 @@
 .PHONY: test bins clean cover cover_ci
-PROJECT_ROOT = go.uber.org/cadence
-
-export PATH := $(GOPATH)/bin:$(PATH)
-
-THRIFT_GENDIR=.gen
 
 # default target
 default: test
+export PATH := $(GOPATH)/bin:$(PATH)
 
-# define the list of thrift files the service depends on
-# (if you have some)
-THRIFTRW_SRCS = \
-  idl/github.com/uber/cadence/cadence.thrift \
-  idl/github.com/uber/cadence/shared.thrift \
```

Comment:
generating `cadence.thrift` and then `shared.thrift` led to `shared.thrift`-generated files overwriting the `cadence.thrift`-generated shared files... which are exactly the same, but it ends up triggering a re-build every time because the files are "too new" for `cadence.thrift` targets.

so since it's just wasted effort and it leads to more wasted effort, I collapsed it to just the one thrift file.